docs/development.md

```diff
@@ -133,6 +133,6 @@ go build .
 
 #### Windows CUDA (NVIDIA)
 
-In addition to the common Windows development tools described above, install:
+In addition to the common Windows development tools described above, install CUDA **AFTER** you install MSVC 2019.
```

Comment:
Similar, will any cuda sdk work for development purposes?

Reply:
It should in theory.

Reply:
Confirmed, I can soften the version checking to accept any MSVC and as long as their CUDA install is aligned, and the right nvcc is in the path, it all works.  (MSVC 2019 + CUDA v11 || MSVC 2022 + CUDA v12)